addons/hr_homeworking/models/res_partner.py

```diff
@@ -1,19 +1,12 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from odoo import models
+from odoo.addons.mail.tools.discuss import Store
 
 
 class ResPartner(models.Model):
     _inherit = 'res.partner'
 
-    def _compute_im_status(self):
-        super()._compute_im_status()
-        for user in self.user_ids:
-            dayfield = self.env['hr.employee']._get_current_day_location_field()
```

Comment:
How comes the new compute doesn't use _get_current_day_location_field like before?